src/axom/slic/docs/sphinx/sections/synopsis.rst

```diff
@@ -0,0 +1,59 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+Slic is a component of the `Axom Toolkit`_, currently in development at
+`Lawrence Livermore National Laboratory (LLNL) <http://www.llnl.gov>`_.
+Slic provides a *light-weight*, *modular* and *extensible* logging
+infrastructure that simplifies logging application messages.
+
+**Key Features**
+
+Some of the features of Slic include the following:
+
+* :ref:`sections/slic_macros` to streamline and simplify logging.
+
+* Designed for interoperability across the constituent libraries of an
+  application. Messages logged by an application and any of its libraries
+  using Slic have a unified format and routed to a centralized output
+  destination.
+
+* Flexible. The :ref:`logMessageFormat` can be customized to suit the
+  application requirements.
+
+* Extensible. Custom handling and filtering of log messages is supported by
+  extending the :ref:`logStream` base class.
+
+* :ref:`NativeLogStreams` to support common logging use cases, e.g., log to
```

Comment:
Not sure what "Native" means here. Would a different adjective in the description be better?